<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistogramSpecification</class>
 <widget class="QWidget" name="HistogramSpecification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0">
      <item row="1" column="0">
       <widget class="QLabel" name="functionLabel">
        <property name="text">
         <string>Function</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="refFuncComboBox">
        <item>
         <property name="text">
          <string>Linear</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="refImgName">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="refImgLabel">
        <property name="text">
         <string>Reference Image: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="loadRefImgButton">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="equalizationCheckbox">
        <property name="text">
         <string>Equalization</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="applyButton">
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
